syntax = "proto3";

import "google/protobuf/timestamp.proto";

message SessionHistory {
  int64 user_id = 1;
  int64 track_id = 2;
  int64 total_chunks_sent = 3;
  int64 total_chunks = 4;
  google.protobuf.Timestamp timestamp = 5;
}